### DISCUSSION TITLE
I'm not able to use `JUST`

### DISCUSSION THREAD

**Discussion by ShubhSensei** [ethereum-optimism/developers#499] [Application Development] [answered]
- OS: Windows -> WSL 
- I installed `JUST` using `cargo install just` and it installed successfully
- Later when I type the command `just install-contracts`, it gives me the following error

Error
```error
Command 'just' not found, did you mean:
  command 'dust' from snap dust (v1.0.0)
  command 'joust' from snap joust (1.0)
  command 'jest' from deb jest (27.5.1~ds+~cs69.51.22-2)
See 'snap info <snapname>' for additional versions.
```
Possible Reason

```
By default, Cargo installs packages into a central location managed by Cargo itself. This is typically in your user's home directory (e.g., `~/.cargo/bin` on Linux/macOS or `%USERPROFILE%\.cargo\bin` on Windows).
```

Can anyone tell me, how to solve this?

**Comment by mds1** [comment, marked as answer]
You just need to add the directory of cargo binaries to your PATH. For example (it might be different on windows):
```sh
export CARGO_PATH="$HOME/.cargo"
export PATH="$PATH:$CARGO_PATH/bin"
```

**Comment by tynes** [comment]
@ShubhSensei thanks for being interested in contributing to optimism. This repo is for technical spec discussion, asking about `just` or other dev questions is better for https://github.com/ethereum-optimism/developers/discussions
